src/sbt-test/sbt-typesafe-conductr/sbt-typesafe-conductr-tester/version.properties

```diff
@@ -1 +0,0 @@
-../../../../version.properties
```

Comment:
Sorry for not feeding back during the review. I want to get your PR in and then tidy-up. 

> On 3 Apr 2015, at 11:49 pm, Konrad Malawski <notifications@github.com> wrote:
> 
> In src/sbt-test/sbt-typesafe-conductr/sbt-typesafe-conductr-tester/version.properties:
> 
> > @@ -1 +0,0 @@
> > -../../../../version.properties
> I was not aware this is possible in scripted tests:
> 
> addSbtPlugin("com.typesafe.conductr" % "sbt-typesafe-conductr" % sys.props("project.version"))
> Would be great if this would have been pointed out during review, would have learnt something about scripted right away :)
> 
> —
> Reply to this email directly or view it on GitHub.
>